environment/src/main/java/com/sequenceiq/environment/environment/validation/securitygroup/AwsEnvironmentSecurityGroupValidator.java

```diff
@@ -31,7 +32,7 @@ public void validate(EnvironmentDto environmentDto, ValidationResult.ValidationR
             if (onlyOneSecurityGroupIdDefined(securityAccessDto)) {
                 resultBuilder.error(securityGroupIdsMustBePresent());
             } else if (isSecurityGroupIdDefined(securityAccessDto)) {
-                if (!Strings.isNullOrEmpty(environmentDto.getNetwork().getNetworkCidr())) {
+                if (RegistrationType.CREATE_NEW.equals(environmentDto.getNetwork().getRegistrationType())) {
```

Comment:
we need logs here, since the entire validation does not contain even a single line of log, and I was not able to find the error based on logs just with repro the case